footer.php

```diff
@@ -1,9 +1,36 @@
 
       <?php wp_footer(); ?>
 
-    <!--<footer>
-      <p>This is our Footer, Here!</p>	
-    </footer>-->
+    <footer class="footer">
+    	<ul>
+	      	<li class="footer-icons">
+	      		<a href="https://www.facebook.com">
+	      			<img src="<?php echo get_stylesheet_directory_uri(); ?>/images/setupshop_ftfacebook1.png" />
+      			</a>
+  			</li>
```

Comment:
This is odd, on sublime everything is properly aligned, on github the alignment shifts and I don't know what github expects me to do. Could github have a bug of sort? It looks perfectly fine on sublime